### ISSUE TITLE
[CRITICAL] Error handling incoming event in hipchat.py

### ISSUE THREAD

**Issue by ptomkiel-oktawave** [skoczen/will#338]
Full log attached below:
```
[CRITICAL] Error handling incoming event {'namespace': 'jabber:client', '_index': 0, 'xml': <Element '{jabber:client}message' at 0x7f0baafd0688>, 'plugins': OrderedDict(), 'loaded_plugins': set(), 'iterables': [], 'tag': '{jabber:client}message', 'parent': None, 'type': 'groupchat', 'from': 'xyz@conf.hipchat.com/Link', 'mid': 'xyz', 'to': 'xyz@chat.hipchat.com/bot||proxy|pubproxy-d100.hipchat.com|1234', 'ts': '1234', 'xmpp_jid': 'xyz@conf.hipchat.com', 'body': '\n\n\n<p><b><a href="https://www.sudosatirical.com/articles/man-loses-will-to-live-during-gentoo-install/">Man Loses Will to Live During Gentoo Install</a></b></p>\n\n\n<p>A man has been left hospitalised and his family left distraught in the aftermath of a grueling install of the Gentoo Linux operating system. The 32 year old man was admitted on Thursday and is currently in a stable but serious condition, after he reportedly lost the will to live.</p>\n\n'}: 
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/will/backends/io_adapters/base.py", line 43, in handle_incoming_event
    m = self.normalize_incoming_event(event)
  File "/usr/lib/python3.6/site-packages/will/backends/io_adapters/hipchat.py", line 644, in normalize_incoming_event
    if sender.id == self.me.id:
AttributeError: 'NoneType' object has no attribute 'id'
```

**Comment by skoczen** [comment]
@ptomkiel-oktawave thanks for the report - this will be fixed in 2.1.1!

**Comment by skoczen** [comment]
This has been fixed, and released in 2.1.1!
